Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+public sealed class PlayerMovement : MonoBehaviour

```

Comment:
Heb verder niet echt een comment naast dat ik me afvraag of jullie hebben nagedacht over een base Movement class waarvan zowel player als enemy kan overerven? 

Een mooi iets om hier bijvoorbeeld in te plaatsen is een protected Rigidbody variable die in de awake/start met een getcomponent wordt opgevraagd. Is gewoon een gedachte though en hoef je verder niks mee te doen :)

Reply:
Zou kunnen, maar geen idee of de speler een statemachine gaat gebruiken. Goed idee, maar toepassing misshien iets te veel, vooral omdat de enemy movement heel anders wordt.